config.defaults.yaml

```diff
@@ -105,3 +105,6 @@ docs:
     srv:
       coordinates: [330.302266, 66.153712]
       title: Semi Regular Variable
+    variable:
+      coordinates: [272.8552844, 29.9028479]
```

Comment:
This is a wrong example - it's an artifact near a bright saturated star (see the attached image)
![screenshot](https://user-images.githubusercontent.com/34982123/119957735-a3e17100-bfa2-11eb-9d78-4dd4f4f5a4f9.png)